### ISSUE TITLE
My Suggestions on improving the server.

### ISSUE THREAD

**Issue by SammySapphire** [Hazbelll/EquestrianHorizon#3]
- Give Changeling King battering ram or increase the speed it takes to lockpick a door. Raiding Pd takes years with the lockpick, especially if all the doors are locked, either increase the speed of which it takes to lockpick the door or just give a kind a battering ram.

- Royal Medic should have some weapon to defend himself. Royal guards all have at least a glock.
Royal medic is technically a Royal Guard as well, so why do he have no weapons? it doesn't have to be a glock, as this way he will technically be a straight upgrade from stock Guard. Perhaps a knife or fists.

Duh.. that's all that ~kinda bothers me. Probably going to add more here with time, it's just something i've noticed people are complaining about, and would be nice to have ^^

**Comment by thegrb93** [comment]
Royal medic isn't trained in combat like a guard is. And I think changelings have quite enough raiding power already.

**Comment by Hazbelll** [comment]
I agree with @thegrb93, although the Royal Medic already has the ability to use fists. I'm especially against making things easier/quicker for the Changelings as of right now. They **really** don't need it.
